.github/workflows/tests-dashmate.yml

```
@@ -84,14 +84,18 @@ jobs:
         run: echo "sha=$(git log -1 --format="%h" -- packages/dashmate)" >> $GITHUB_OUTPUT
         if: inputs.restore_local_network_data == true
 
+      - name: Set HOME variable to github context
+        shell: bash
+        run: echo "HOME=$HOME" >> $GITHUB_ENV
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:22: Double quote to prevent globbing and word splitting

(shellcheck)